lib/utils/oclif-utils.ts

```diff
@@ -69,3 +69,28 @@ export function capitanoizeOclifUsage(
 		.replace(/(?<=\s)[A-Z]+(?=(\s|$))/g, (match) => `<${match}>`)
 		.toLowerCase();
 }
+
+export async function getCommandsFromManifest() {
+	const fs = await import('fs');
+	let manifest;
+	try {
+		manifest = JSON.parse(
+			await fs.promises.readFile('./oclif.manifest.json', 'utf8'),
+		);
+	} catch (e) {
+		if (e.code === 'ENOENT') {
+			throw new Error('Oclif manifest not found.');
```

Comment:
Although this should never happen in normal conditions, I suggest rewording the error message a bit:
```
throw new Error(stripIndent`
    ${manifestPath} not found.
    If you are developing the CLI, run \`npm run build\` to fix it.`);
```
... where `manifestPath` is the conditionally determined path for `oclif.manifest.json`.